### PR TITLE
[WIP] Update Content to Ampersand.md, Greater-And-Less-Than.md and Non-Brea…

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
…king-Space.md 

[WIP] HTML additions

Hey, @lizTheDeveloper I have updated/edited the content portion of these insights. However, I need someone to look over the changes and tell me if I am being too wordy, or not wordy enough. Also, I can't get a picture to show in the nk-md Enki app to test how it looks. Should I delete that and just explain the picture portion through text? Lastly, should I update the author, like I did in the 2 insights, or leave it as mihaiberq, as I did in one of them?

### CODE REVIEW

**Comment by enki-hq** [comment]
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/Web/HTML/html-entities/ampersand.md?ref=36b169bee6295a954cac8e782c14a52e7a360a51
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/Web/HTML/html-entities/greater-and-less-than.md?ref=36b169bee6295a954cac8e782c14a52e7a360a51
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/Web/HTML/html-entities/non-breaking-space.md?ref=36b169bee6295a954cac8e782c14a52e7a360a51

**Comment by lizTheDeveloper** [comment]
Hey @Stefan-Stojanovic, just reviewed these, sorry for the delay.
These are absolutely excellent, very high quality writing. I think you'll be an excellent content contributor! 💯 🥇 🎉 

The next step is to make sure you can write questions.
Most of the questions (the ones that were pre-written) in those particular insights are almost ready to add, but I think the formatting and potentially the writing needs to be reviewed and fixed.
Can you take a look? Some of them need some additional incorrect (but plausible) answers added, others just need to be cleaned up a bit.

Here's the documentation for questions: https://github.com/enkidevs/curriculum/wiki/Insight-Documentation#questions
Let me know if that makes it clear how to proceed, or if I need to update those docs 😄 (you are one of the early users of those docs, so your feedback is super valuable!)

**Comment by lizTheDeveloper** [comment]
hey @Stefan-Stojanovic - I merged in another branch where many of the questions were fixed, but there are still some that aren't properly formatted on the `html-course-2` branch, so target those 👍

**Comment by enki-hq** [comment]
Must have exactly 1 yaml configuration but found 0 instead.

**Comment by enki-hq** [comment]
Must have exactly 1 yaml configuration but found 0 instead.

**Comment by enki-hq** [comment]
Must have exactly 1 yaml configuration but found 0 instead.
